### PR TITLE
Annual steez

### PR DESCRIPTION
**Pull request by emilyville**
- added a base stylesheet for SCI (a pretty basic start but eventually it will be amazing)
- added instrument boxes - this would create a slight change in process to get to the assessments, and I haven't totally figured that out yet. For now the result input field is just hidden, but for some shorter ones like SWLS we should be able to expand below and fit them in the box
- added placeholder text and more verbose labels where it seemed appropriate.

Here's where I'm headed in a rough sense, will make pr for patient header next

annual evaluation copy

### CODE REVIEW

**Comment by awong-dev** [comment]
@emilyville Fix travis error?

File: app/assets/stylesheets/sci.css.scss [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+.panel-form {	
+	h4, .panel-title {
+    	color: #406e7b;
+    	size: 14px;
+    	margin: 10px;
+    }
+    .panel-body {
+    	padding: 40px;
+    }
+	.panel-title {
```
Comment (awong-dev) [inline app/assets/stylesheets/sci.css.scss]: nit: Indentation in this file seems inconsistent. fix?

File: app/views/annual_evaluations/_form.html.slim [hunk 2]
```diff
@@ -1,33 +1,44 @@
...
+    = f.input :neuro_zone_of_preservation3, label: false, :placeholder => "00"
 
-    h2 ASIA and LOI
-    = f.input :level_of_injury
-    = f.input :asia_score
+    h4 ASIA &mdash; LOI
+    .instrument
+        h4
+          | ASIA-LOI: A-C6  
+          span.small = link_to "view record: 12/23/1987"
```
Comment (awong-dev) [inline app/views/annual_evaluations/_form.html.slim]: Why is there a hard coded date? Should there be a TODO + a issue filed for this?

File: app/views/annual_evaluations/_form.html.slim [hunk 3]
```diff
@@ -1,33 +1,44 @@
...
+            = link_to "+ add new assessment", new_patient_omr1_year_path(@patient)
+        / = f.input :level_of_injury
+        / = f.input :asia_score
 
-    h2 Administrative
-    = f.input :eval_offered, label: "Date Offered", as: :string, input_html: { class: "datepicker" }
-    = f.input :eval_completed, label: "Date of Completion", as: :string, input_html: { class: "datepicker" }
-    Display next AE Due here.
+    h4 Administrative
+    = f.input :eval_offered, label: "Date Offered", :placeholder => "June 10, 2015", as: :string, input_html: { class: "datepicker" }
```
Comment (awong-dev) [inline app/views/annual_evaluations/_form.html.slim]: FYI, I hate our date picker. would love to talk options :)

Comment (awong-dev) [inline app/views/annual_evaluations/_form.html.slim]: Should we use the current date for a place holder?  And does this understand the date format June 10, 2015? I thought we needed mm/dd/yyyy

File: app/views/annual_evaluations/_form.html.slim [hunk 4]
```diff
@@ -1,33 +1,44 @@
...
+    
     = f.input :is_inpatient
 
-    h2 Standard Instruments
-    = f.input :cyh
-    = f.input :fim
-    = f.input :kurtzke_edss
+    h4 Standard Instruments
+    .instrument
+        h4 = link_to "+ add CYH assessment", new_patient_omr1_year_path(@patient)
```
Comment (awong-dev) [inline app/views/annual_evaluations/_form.html.slim]: Can we link to something like edit_patient_path(@patient) or just '#' even?

Using this specific block of no-op code will end up causing more work when we delete omr1_year.

**Comment by awong-dev** [comment]
Mostly style nits. One functionality comment. Then gotta keep travis green.

Let me know when it's ready for another pass!

File: app/views/annual_evaluations/_form.html.slim (hunk 2 above)
Comment (emilyville) [inline app/views/annual_evaluations/_form.html.slim]: The link also doesn't link to anything! 

I will add a TODO for all the things it needs to eventually do

**Comment by awong-dev** [comment]
LGTM!

**Pull request merged**